app/frontend/src/features/auth/login/UsernameForm.tsx

```diff
@@ -57,7 +56,7 @@ export default function UsernameForm() {
           break;
       }
     } catch (e) {
-      dispatch(authError(e.message));
+      authActions.authError(e.message);
     }
     setLoading(false);
```

Comment:
I'm quite consistently getting that setState on unmounted component error on this now. That isMounted "hack" may actually be useful here...